network/src/proto/consensus.proto

```diff
@@ -102,16 +102,10 @@ message BlockInfo {
 }
 
 message VoteData {
-  // The id of the block being vote for.
-  bytes block_id = 1;
-  // The id and the version of the state after executing the block.
-  bytes executed_state_id = 2;
-  // The round of the block being voted for
-  uint64 round = 3;
-  // The id of the parent block
-  bytes parent_block_id = 4;
-  // The round of the parent block
-  uint64 parent_block_round = 5;
+  // Contains all the block information needed for voting for the proposed round.
+  BlockInfo proposed = 6;
```

Comment:
You probably want to reset the numbering back to 1 and 2.

Reply:
That works, I had been only appending rather than overwriting proto files. Though there's no reason for that in this case (and many of our cases) as we don't really support backwards compatibility.